models/_sector/dex/trades/dex_stg_trades.sql

```diff
@@ -0,0 +1,54 @@
+{{ config(
+    schema = 'dex'
+    , alias = 'stg_trades'
+    , materialized = 'view'
```

Comment:
@grkhr i think one reason to materialize this level is because each blockchain-specific spell below in the list are views that join to `<blockchain>.transactions` base table to grab `tx_from`/`tx_to` columns. rather than materialize each one of those, we can materialize here once. then the final stage will materialize with further logic, such as metadata joins and column metrics.

here is the line to join to base table per blockchain:
https://github.com/duneanalytics/spellbook/pull/4533/files#diff-9c7ba2ea0d536632d3c6b2c9a7483c8dbf03d497288f35620ac02991e5b946e5R42

Reply:
Makes sense! Just looked again through all of the logic. Let's leave step 2 materialized.

Reply:
I was thinking that project-chain-specific spells should materialize tx_from/to, but realized that it will be inconvenient to the end contributors. Always better to keep requirements to the end users simple.